applications/API/serializers.py

```
@@ -0,0 +1,487 @@
+                return None
+            elif obj.added_by.first_name and not obj.added_by.last_name:
+                return f"{obj.added_by.first_name}"
+            elif not obj.added_by.first_name or not obj.added_by.last_name:
+                return f"{obj.added_by.username}"
+            return f"{obj.added_by.first_name} {obj.added_by.last_name[0]}."
+        except IndexError:
+            return "Unknown User."
+
+    class Meta:
+        model = Insult
+        fields = ["id", "content", "category", "status", "nsfw", "added_by", "added_on"]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)